caracal/workers/transform_worker.py

```diff
@@ -63,7 +63,7 @@ def worker(pipeline, recipe, config):
         field_to_split = config['split_field']['field'].split(',')
 
         if 'calibrators' in field_to_split:
-            field_to_split = ['fcal','bpcal','gcal']
+            field_to_split = ['fcal','bpcal','gcal','xcal']
 
         for fd in field_to_split:
             if fd not in ['target','fcal','bpcal','gcal']:
```

Comment:
I am adding 'xcal' here, in keeping with Line 66 just above, otherwise the pipeline complains. If that's not what you intended, please revert.

Reply:
Oh shit, how did that sneak through? Anyway, your fix is the correct one.

Reply:
I'm still confused about why we're doing this now. We don't do anything with xcal, we don't apply crosscal to it and we don't solve for polcal terms. Why don't we include xcal back into the pipeline in a fully consistent way once we have a functional polcal worker again? I know we've failed to converge on this point before -- maybe there's a simple answer that I've failed to understand so far.

Reply:
I think it's being done in the spirit of doesn't cost anything, doesn't do anything (or does this mean we're suddenly going to have a spate of useless xcal plots? I assume ``pipeline['xcal']`` is an empty list at the moment, right?), and it's one fewer potential bug to look for later.

Reply:
Well, if we want to take out xcal, then let's take it out of both the lines, otherwise it's just going to crash. @o-smirnov, what do you think? To xcal or not to xcal?

Reply:
no, I've got datasets where pipeline['xcal'] is not empty

Reply:
Ah, I missed @o-smirnov 's comment above before making my own, sorry. @paoloserra , just to clarify, you have extra plots with xcal? (I think xcal defaults to bpcal, if nothing is given?)

Reply:
I don't know whether I get extra plots, at the moment I don't of course, which I'm happy with. Anyway guys, do what you want about this My opinion is that we put xcal back into the pipeline in one go instead of bits and pieces, but I will stop making a big deal about it. :)

Reply:
How about we take it out, but leave an ``# NB: xcal needed`` comment behind?